src/styles/header.scss

```
@@ -0,0 +1,114 @@
+.header {
+  height: 100vh;
+  padding: 0 $s-pad;
+
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 2px;
+
+  background-image: url(../images/header.jpg);
+  background-size: cover;
+  background-position: center;
+
+  &__logo {
+    position: absolute;
+    width: $logo-w;
+    height: $logo-h;
+    left: calc(50% - #{$logo-w/2});
+  }
+
+  &__nav {
+    bottom: 48px;
+    position: absolute;
+  }
+
+  &__title {
+    position: absolute;
+    bottom: 150px;
+    margin: 0;
+
+    display: block;
+
+    font-weight: 700;
+    font-size: 60px;
+    line-height: 60px;
+    letter-spacing: -1.5px;
+    color: #fff;
+
+    &::before {
+      position: absolute;
+      top: -16px;
+      height: 2px;
+      width: 35px;
+
+      content: "";
+
+      background-color: #fff;
+      border-radius: 2px;
+    }
+  }
+
+  &__top {
+    position: relative;
+    top: 24px;
+
+    display: flex;
+    justify-content: space-between;
+
+    font-weight: bold;
+    font-size: 12px;
+    line-height: 16px;
+  }
+
+  &__phone-num,
+  &__action {
+    text-transform: uppercase;
+    color: #fff;
+    position: relative;
+    top: 40px;
+    display: block;
+    transition: color $time, transform $time ease;
```

відсортуй по категоріям
```suggestion
    position: relative;
    top: 40px;
    
    display: block;

    color: #fff;
    text-transform: uppercase;
    
    transition: color $time, transform $time ease;
```